### ISSUE TITLE
R1 - Site Manager Roles not fully defined or implemented.

### ISSUE THREAD

**Issue by mancusop** [ccswbs/hjckrrh#309]
@mmafe @anncesar @jpinsent @enichol 
In Feature Dev Site Manager is defined as having the following permissions:
- Can create new users.
- Can edit any user profile.

The previous points are still not currently implemented? 

It is not defined if they get access to blocks/menus/taxonomy but that access is given to them in current Site Manager role. Should R1 be updated with these permissions?

Are Site Managers supposed to have all the permissions of roles below them like Author and Editor? They currently lack access to any content creation/editing/viewing.

**Comment by jpinsent** [comment]
The site manager not being able to interact with user profiles is an issue that was demonstrated during the last demo session. The permissions exist, but there is no user interface for the site manager to access these profiles. I recall Brent mentioning that we may have to create our own interface, since the existing one ends up giving the site manager too many permissions. Either way, it's an issue we need to address at some point. I think this is the Trello card associated with this issue: https://trello.com/c/4F3bcLaO

I'm not sure about Site Managers having the same permissions as Authors and Editors. We need a more thorough specification that outlines everything that the Site Manager is intended to have access to. Currently, this is the only documentation I can find on the role: https://trello.com/c/Mw585OO3

**Comment by mancusop** [comment]
Yup - clearer spec needed as It is difficult for me to write what Site Managers can or cannot do. ex. Are they responsible for Taxonomy? Menu creation? etc...

**Comment by mancusop** [comment]
@mmafe @mmatthews @brharp Just a heads up - when a site manager is created they get very little priviliages - however they can add other roles to themselves in their user info page that give them abilities to add content (they originally cant)

They still however cannot add users.

**Comment by mmafe** [comment]
This issue has been solved.
